### PR TITLE
chore: update cta to include celo round

### PR DESCRIPTION
**Pull request by kelsos**

### CODE REVIEW

File: messages/dashboard.json [hunk 1]
```diff
@@ -34,5 +34,17 @@
       "start": 1722981600,
       "end": 1724191200
     }
+  },
+  {
+    "message": "Enjoying rotki? Consider ",
+    "message_highlight": "supporting us in",
+    "action": {
+      "text": "Real World Builders Round at Gitcoin",
```
Comment (LefterisJP) [inline messages/dashboard.json:42]:
```suggestion
      "text": "Celo Real World Builders Round at Gitcoin",
```

File: messages/dashboard.json [hunk 2]
```diff
@@ -34,5 +34,17 @@
       "start": 1722981600,
       "end": 1724191200
     }
+  },
+  {
+    "message": "Enjoying rotki? Consider ",
+    "message_highlight": "supporting us in",
```
Comment (LefterisJP) [inline messages/dashboard.json:40]: Why in the dashboard we are missing "the" but in welcome.json we show it?

File: messages/welcome.json [hunk 3]
```diff
@@ -38,6 +38,19 @@
         "start": 1722981600,
         "end": 1724191200
       }
+    },
+    {
+      "header": "Enjoying rotki?",
+      "icon": "https://raw.githubusercontent.com/rotki/data/develop/messages/icon.svg",
+      "text": "rotki is part of the Real World Builders Round at Gitcoin",
```
Comment (LefterisJP) [inline messages/welcome.json:45]:
```suggestion
      "text": "rotki is part of the Celo Real World Builders Round at Gitcoin",
```

File: messages/dashboard.json (hunk 2 above)
Comment (kelsos) [reply messages/dashboard.json:40]: I remember we had a couple of cases where the didn't make sense in the highlight and this time I just reused the template.

Comment (LefterisJP) [reply messages/dashboard.json:40]: Can you add "the" in the new CTAs that make sense? The message highlight is the link itself?

Comment (kelsos) [reply messages/dashboard.json:40]: Yes

Comment (kelsos) [inline messages/dashboard.json:40]:
```suggestion
    "message_highlight": "supporting us in the",
```

**Pull request merged**